src/components/Login/helpers/tokenService.tsx

```diff
@@ -0,0 +1,51 @@
+export default function userTokenService() {
+  function getLocalRefreshToken() {
+    if (typeof window !== "undefined") {
+      const userToken = JSON.parse(window.localStorage.getItem("userToken") as string);
+      return userToken.refreshToken;
+    }
+  }
+
+  function getLocalAccessToken() {
+    if (typeof window !== "undefined") {
+      const userToken = JSON.parse(window.localStorage.getItem("userToken") as string);
+      return userToken.accessToken;
+    }
+  }
+
+  function updateLocalAccessToken(accessToken: string) {
+    if (typeof window !== "undefined") {
+      const userToken = JSON.parse(window.localStorage.getItem("userToken") as string);
+      userToken.accessToken = accessToken;
+      window.localStorage.setItem("userToken", JSON.stringify(userToken));
+    }
+  }
+
+  function removeUser() {
+    if (typeof window !== "undefined") {
+      window.localStorage.removeItem("userToken");
+    }
+  }
+
+  function getUserToken() {
+    if (typeof window !== "undefined") {
+      const userToken = JSON.parse(window.localStorage.getItem("userToken") as string);
+      return userToken;
+    }
+  }
+
+  function setUserToken(userToken: object) {
+    if (typeof window !== "undefined") {
+      window.localStorage.setItem("userToken", JSON.stringify(userToken));
+    }
+  }
+
+  return {
+    getLocalRefreshToken,
+    getLocalAccessToken,
+    updateLocalAccessToken,
+    removeUser,
+    getUserToken,
+    setUserToken,
+  };
+}
```

Comment:
You seem trying to make a custom hook but there are no nooks in it (no react)
Maybe simply export all functions individually and import them as you need.